vaadin-components-gwt/src/main/webapp/vaadin-grid/vaadin-grid.html

```diff
@@ -779,19 +779,26 @@
           this.contentDocument.defaultView.addEventListener('resize', function(e) {
             _this._grid.redraw();
           });
-          _this._grid.redraw();
+
+          _this.then(function() {
```

Comment:
To me is clearer to use _this only in inner functions.  So use here this preferebly